sound/soc/sof/loader.c

```diff
@@ -127,6 +127,20 @@ int snd_sof_fw_parse_ext_data(struct snd_sof_dev *sdev, u32 bar, u32 offset)
 }
 EXPORT_SYMBOL(snd_sof_fw_parse_ext_data);
 
+static int ext_man_get_fw_version(struct snd_sof_dev *sdev,
+				  const struct sof_ext_man_elem_header *hdr)
+{
+	const struct sof_ext_man_fw_version *v;
+
+	v = container_of(hdr, struct sof_ext_man_fw_version, hdr);
```

Comment:
const struct sof_ext_man_fw_version *v =
	container_of(hdr, struct sof_ext_man_fw_version, hdr);

.. would seem to fit nicer.

Reply:
chnaged